### PR TITLE
RHIDP-4409: update metric monitoring instructions for OCP and AKS

### PR DESCRIPTION
**Pull request by JessicaJHee**
**IMPORTANT: Do Not Merge - To be merged by Docs Team Only**

**Version(s): 1.4**

Add the relevant labels to the Pull Request.
**Issue: [RHIDP-4409](https://issues.redhat.com/browse/RHIDP-4409)**

### CODE REVIEW

File: modules/observe/proc-rhdh-monitoring-logging-aks.adoc [hunk 1]
```diff
@@ -3,19 +3,88 @@
...
 
 By default, Prometheus uses the minimum ingesting profile, which optimizes ingestion volume and sets default configurations for scrape frequency, targets, and metrics collected. The default settings can be customized through custom configuration. Azure offers various methods, including using different ConfigMaps, to provide scrape configuration and other metric add-on settings. For more information about default configuration, see https://learn.microsoft.com/en-us/azure/azure-monitor/containers/prometheus-metrics-scrape-default[Default Prometheus metrics configuration in Azure Monitor] and https://learn.microsoft.com/en-us/azure/azure-monitor/containers/prometheus-metrics-scrape-configuration?tabs=CRDConfig%2CCRDScrapeConfig[Customize scraping of Prometheus metrics in Azure Monitor managed service for Prometheus] documentation.
 
+== Configuring annotations for monitoring
```
Comment (themr0c) [inline modules/observe/proc-rhdh-monitoring-logging-aks.adoc:21]: Consider splitting in 2 procedure modules

Comment (hmanwani-rh) [reply modules/observe/proc-rhdh-monitoring-logging-aks.adoc]: Done

**Review by zdrapela — APPROVED** [review]
AKS is looking good 👍

File: modules/observe/proc-view-logs-aks.adoc [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+.Prerequisites
+
+* You have deployed {product-short} on {aks-short}.
+
+For more information, see xref:{installing-on-aks-book-url}#assembly-install-rhdh-aks[{installing-on-aks-book-title}].
+
+.Procedure
+
+View live logs from your {product-short} instance::
```
[the remark this answers is not in the document]
Comment (themr0c) [reply modules/observe/proc-view-logs-aks.adoc:14]: Got it, we have 2 disticnt procedures here:

* Viewing live logs from your {product-short} instance
* Viewing live logs from the Container Engine

[the remark this answers is not in the document]
Comment (themr0c) [reply modules/observe/proc-view-logs-aks.adoc:14]: @hmanwani-rh agreed to work on it later.

**Comment by JessicaJHee** [comment]
@themr0c @zdrapela Thank you for the review!

**Pull request merged**